docs/DeveloperGuide.adoc

```diff
@@ -16,24 +16,31 @@
 . Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
 . Set up the correct JDK version
 .. Click `Configure` > `Project Defaults` > `Project Structure`
-.. If JDK 8 is listed in the drop down, select it. If it is not, click `New...` and select the directory where you installed JDK 8.
+.. If JDK 9 is listed in the drop down, select it. If it is not, click `New...` and select the directory where you installed JDK 9.
 .. Click `OK`.
 . Click `Import Project`
 . Locate the project directory and click `OK`
 . Select `Create project from existing sources` and click `Next`
 . Rename the project if you want. Click `Next`
 . Ensure that your `\src` and `\test\java` folders are checked. Keep clicking `Next`
 . Click `Finish`
+. Try running the application (right-click the `Main` class under the `src` folder, and click `Run 'Main.main()'`)
+. Observe how the application fails to compile. This is because the `JAXB` APIs is no longer contained under the default classpath from Java 9 onwards. To fix this issue, the `JAXB` module needs to be added to the classpath:
+.. Open any of the files which has the compilation error (e.g `StorageFile.java`), and place your cursor over any `javax.xml.bind` in the imports highlighted in red
+.. Press ALT+ENTER and select `--add-modules=java.xml.bind to module compiler options`
+.. Go to `Run` -> `Edit Configurations...`
+.. Under `Configuration`, add `--add-modules java.xml.bind` into `VM options`
+. Run the application again to ensure it compiles and runs without any errors.
```

Comment:
Remove the full stop at the end of this sentence, to be consistent with the rest of the list items.